modules/proxy/variables.tf

```diff
@@ -0,0 +1,50 @@
+variable "cluster_assistant_port" {
+  type        = number
+  description = "The port of the Cluster Assistant."
+  default     = 23010
+}
+
+variable "k8s_api_port" {
+  type        = number
+  description = "The port of the Kubernetes API."
+  default     = 6443
+}
+
+variable "install_id" {
+  type        = string
+  description = "Identifier for install to apply to resources"
+}
+
+variable "ports" {
+  type        = list(number)
+  description = <<-EOD
```

Comment:
was there a reason for using the EOD instead of just a string?

Reply:
Just to avoid line wrapping in editors. :blush: Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/50/commits/e1d44b6fb9880ce28f587c90b9cac8f60a143989.